README.md

```diff
@@ -50,6 +50,7 @@
 
```

Comment:
| 🔴 Your Pull Request Summary | 🟢 Descriptive Pull Request Summary |
| --- | --- |
| ![image](https://github.com/HtetWaiYan7191/testing-todolist/assets/93039359/54bec128-cdc9-4ebe-beea-9d3e920c3e8c) | ![image](https://github.com/HtetWaiYan7191/testing-todolist/assets/93039359/8fcf73cc-3852-4cb2-9df5-407268e67108) |

- [ ] It would be nice If you make your pull request summary more descriptive by adding more details about the implementations you have done in this pull request 👍